test/Signature.ts

```diff
@@ -240,86 +255,39 @@ describe('Signature', function () {
       [messageWindow] = await waitForNewWindows(1);
       await browser.switchToWindow(messageWindow);
       await browser.refresh();
-    }
-
-    async function getPermissionRequestResult() {
-      return JSON.parse(
-        await browser.execute(() => {
-          const { result } = window;
-          delete window.result;
-          return result;
-        })
-      );
-    }
-
-    after(async function () {
```

Comment:
Does calling `navigateTo` in every test solves some problem? It can take a lot of time, that's why I avoided doing it in every test below

Reply:
done